service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java

```diff
@@ -715,4 +800,28 @@ public void onMessage(Message msg, Object message) {
             }
         });
     }
+
+    private void setupClinicalSampleUpdateHandler(Gateway gateway,
+            MessageHandlingService messageHandlingService) throws Exception {
+        gateway.subscribe(DMP_SAMPLE_UPDATE_TOPIC, Object.class, new MessageConsumer() {
+            @Override
+            public void onMessage(Message msg, Object message) {
+                LOG.info("Received message on topic: " + NEW_DMP_SAMPLE_TOPIC);
```

Comment:
forgot to change the topic that's logged [here](https://github.com/mskcc/cmo-metadb/pull/607/files#diff-e1e08df99e843f0281d446ab742c0cb98fdbd395eb2a5904c73059dc827ce160R809) and [here](https://github.com/mskcc/cmo-metadb/pull/607/files#diff-e1e08df99e843f0281d446ab742c0cb98fdbd395eb2a5904c73059dc827ce160R822)